resources/secrets.yaml

```diff
@@ -0,0 +1,23 @@
+kind: Secret
+apiVersion: v1
+metadata:
+  name: kyma-showcase-secret
+  namespace: default
+  selfLink: /api/v1/namespaces/default/secrets/kyma-showcase-secret
+  managedFields:
+    - manager: kubectl-create
+      operation: Update
+      apiVersion: v1
+      fieldsType: FieldsV1
+      fieldsV1:
+        f:data:
+          .: {}
+          f:REDIS_PASSWORD: {}
+          f:GCP_API_KEY: {}
+          f:GCP_EMAIL: {}
+data:
+  GCP_API_KEY: >-
+    {YOUR_GCP_API_KEY}
+  GCP_EMAIL: {YOUR_GCP_EMAIL}
+  REDIS_PASSWORD: {YOUR_REDIS_PASSWORD}
```

Comment:
Do you have any method/technology to inject true values to the fields like this: {YOUR_GCP_API_KEY}?

Reply:
Additionally, if not and you base on tutorials like [this](https://github.com/kyma-incubator/Kyma-Showcase/blob/main/README.md) (installation - step 2) then please add information about this to the README.md. Changes like this should be implemented together with information on how to use it.